lib/ansible/modules/network/eos/eos_config.py

```diff
@@ -328,10 +342,10 @@ def main():
         intended_config=dict(),
 
         # save is deprecated as of ans2.4, use save_when instead
-        save=dict(default=False, type='bool', removed_in_version='2.4'),
+        save=dict(default=False, type='bool', removed_in_version='2.8'),
 
         # force argument deprecated in ans2.2
-        force=dict(default=False, type='bool', removed_in_version='2.2')
+        force=dict(default=False, type='bool', removed_in_version='2.7')
```

Comment:
`removed_in_version` for `force` should be `2.6`. We follow deprecated version + 4 for removed_in_version.

Reply:
Ok, will fix. Does the `removed_in_version` for ios_config need to be changed too then?
https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/ios/ios_config.py#L125
https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/ios/ios_config.py#L413

Reply:
@jmighion would be great if you can update ios_config as well. Thanks!